src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -106,33 +109,30 @@
         :nativeCurrencySymbol   :native-currency-symbol
         :nativeCurrencyName     :native-currency-symbol})))
 
-(defn sort-keypairs
-  [keypairs]
-  (sort-by #(if (some (fn [account]
-                        (string/starts-with? (:path account) constants/path-eip1581))
-                      (:accounts %))
-              0
-              1)
-           keypairs))
-
-(defn sort-and-rename-keypairs
-  [keypairs]
-  (let [sorted-keypairs (sort-keypairs keypairs)]
-    (map (fn [item]
-           (update item
-                   :accounts
-                   (fn [accounts]
-                     (map
-                      (fn [{:keys [colorId] :as account}]
-                        (assoc account
-                               :customization-color
-                               (if (seq colorId)
-                                 (keyword colorId)
-                                 :blue)))
-                      accounts))))
-         sorted-keypairs)))
-
-(defn parse-keypairs
+(defn get-keypair-operability
+  [{:keys [accounts]}]
+  (cond
+    (some #(= (:operable %) :no) accounts)
+    :no
+
+    (some #(= (:operable %) :partially) accounts)
+    :partially
+
+    :else
+    :fully))
```

Comment:
Although the function name is clear enough, I'd suggest renaming the returned keywords to be more explicit, since we may pass these values and it may be confusing.

What about `:partially-operable` o `:partial-operability`?

Reply:
I don't understand what is `get-keypair-operability` doing? What does `partially` operable mean?

Also is the sorting not required?

Reply:
> What about :partially-operable o :partial-operability?

@ulisesmac -  I have no strong opinion as I followed the same value we use in `status-go`. Additionally, I think it's repetitive as the key for that value is `:operable` and we include the same as the suffix. But, I'm happy to change it if everyone felt like that.

Reply:
@smohamedjavid thanks for the clarification. I currently do not see usages of `operable` in the codebase. Is that functionality yet to be implemented?

Reply:
Yes @OmarBasem, it will be used soon in our codebase.
 - To sort the missing keypairs in wallet settings 
 - To show the accounts that are non-operable in different types of account cards in the wallet home

Reply:
@ulisesmac  - As we discussed async about the different keypairs and accounts, I have updated the key name to `lowest-operability` to NOT confuse it with the `operable` key of the accounts.

<hr />

Add a short answer to your question for others to know:

> Do we receive a keypair and its accounts? If so, consider adding :as _keypair to the destructuring.

Yes, each keypair map will have an `accounts` key which has all the accounts generated from it.

> If so, then each account inside a keypair has an operability defined? why? I don't understand it, I believe all the accounts inside a keypair have the same operability as the keypair, since they are derived from it.

It depends on the keypair. 

Let's take Profile keypair. If you recover the account, the default account is fully operable. 
But, if there is any other account generated from it, it will be partially operable for security reasons. https://github.com/status-im/status-desktop/issues/10772

If there is any seed-phrase key pair or private key - key pair, all the accounts generated from it are `no` operable as its keystore is missing.

Fully -> Keystore files are there and validated
Partially -> Keystore files are there but need to be validated (by the user's password)
No -> There are no keystore files for this keypair

For Partially operable accounts/keypair -> It looks the same as the fully operable to the user. Whenever there is password authentication in the app, we need to call one other method to make partially operable accounts fully operable.

Introducing this new key `lowest-operability` will help us sort the missing keypairs quickly in the wallet settings.